### ISSUE TITLE
Hardcoded makeLegacyMessageCode() limit

### ISSUE THREAD

**Issue by matteotrubini** [wintercms/wn-translate-plugin#26]
https://github.com/wintercms/wn-translate-plugin/blob/7e9f37ffe7266de49c4cfa2d73718670d44da124/updates/v2.1.0/migrate_message_code.php#L75
is ported from
https://github.com/wintercms/wn-translate-plugin/blob/7fa0de32e10f2674e3b7f4f2fa4322ef33bd6fc5/models/Message.php#L309
but it will trown an error in case of MySQL default config with `varcharmax = 191` (see https://github.com/wintercms/winter/blob/ec46549b21893d39c15261aa237e49c9c0680fbb/config/database.php#L70)

I think we need to get limit value using the same logic used here:
https://github.com/wintercms/winter/blob/ec46549b21893d39c15261aa237e49c9c0680fbb/modules/system/ServiceProvider.php#L624-L642
of even better, we can check the current table schema with `Schema::getConnection()->getDoctrineColumn(self::TABLE_NAME, 'code')->getLength()`, but this should be tested against all supported databases to check returned values (I don't have this kind of expertise).

**Comment by LukeTowers** [comment]
@matteotrubini it might be worth pointing out that the varcharmax configuration item is removed in Winter v1.2 since it was a workaround for older version of MySQL / MariaDB that are no longer supported in Winter v1.2 so this may become a non-issue.

**Comment by matteotrubini** [comment]
You are right, but what's happens if someone install on / upgrades an existing DB? As varchar columns were created with that lenght, the limit still apply.
